### PR TITLE
Add link to mailing list

### PR DESCRIPTION
**Pull request by henrik242**
Add info about the https://github.com/spotbugs/discuss/issues mailing list

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -69,7 +69,10 @@
 <h3>
 <a id="support-or-contact" class="anchor" href="#support-or-contact" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Support or Contact</h3>
 
-<p>You can contact us and report issues on <a href="https://github.com/spotbugs/spotbugs/issues">GitHub</a></p>
+<p>You can report issues on <a href="https://github.com/spotbugs/spotbugs/issues">GitHub</a>.</p>
+          
+<p>Or, you can contact us using <a href="https://github.com/spotbugs/discuss/issues?q=">our general purpose mailing list</a>.</p>
```
Comment (iloveeclipse) [inline index.html:74]: What should "?q=" do useful? Shouldn't we just use "https://github.com/spotbugs/discuss/issues"?

Comment (henrik242) [reply index.html:74]: With the `?q=` you see both open and closed issues.

Comment (iloveeclipse) [reply index.html:74]: Cool.

**Review by iloveeclipse — APPROVED** [review]
Thanks.

**Pull request merged**